src/components/dashboard/Relaying/index.tsx

```diff
@@ -50,22 +50,24 @@ const Relaying = () => {
             <Typography color="primary.light" alignSelf="center">
               Transactions per hour
             </Typography>
-            <Box
-              className={classnames(css.relayingChip, {
-                [css.unavailable]: relays && relays.remaining === 0,
-              })}
-            >
-              <SvgIcon component={InfoIcon} fontSize="small" />
-              {relaysError ? (
-                <Typography fontWeight={700}>{limit} per hour</Typography>
-              ) : relays?.remaining !== undefined ? (
-                <Typography fontWeight={700}>
-                  {relays.remaining} of {limit}
-                </Typography>
-              ) : (
-                <Skeleton className={css.chipSkeleton} variant="rounded" />
-              )}
-            </Box>
+            {relays !== undefined ? (
```

Comment:
Nit: I'd suggest keeping unrelated changes like this in separate PRs in the future. This initially confused me.

Reply:
Sorry, I tried to use the PR description to give context to the unrelated changes. Will mind it for future ref